lib/analyze-scope.js

```diff
@@ -131,6 +131,11 @@ class Referencer extends OriginalReferencer {
     super.visitProperty(node);
   }
 
+  CallExpression(node) {
```

Comment:
This is not needed anymore after the Babel upgrade. I'll just leave the test case.